bin/SymfonyPolyfillCleanup.php

```diff
@@ -0,0 +1,234 @@
+
+		foreach ( $list_of_polyfills as $polyfill ) {
+
+			// Search for bootstrap.php
+			$bootstraps = array_unique(
+				$this->find_library_file_pattern( 'bootstrap', $polyfill )
+			);
+
+			foreach ( $bootstraps as $bootstrap ) {
+				$this->remove_statement_with_pattern( $bootstrap, "return require __DIR__.'/bootstrap80.php", 'if (\PHP_VERSION_ID >= 80000)' );
+			}
+
+			// Search for bootstrap80.php
+			$bootstraps80 = array_unique(
+				$this->find_library_file_pattern( 'bootstrap80', $polyfill )
+			);
+
+			foreach ( $bootstraps80 as $bootstrap80 ) {
+				$this->remove_file( $bootstrap80 );
+			}
+		}
+	}
+
+
+	/**
+	 * Remove a statement from a file.
+	 *
+	 * @param string $file
+	 * @param string $inner_pattern The inner pattern for the  statement.
+	 * @param string $statement_pattern The statement pattern.
+	 */
+	protected function remove_statement_with_pattern( string $file, string $inner_pattern, string $statement_pattern ) {
+		if ( ! file_exists( $file ) ) {
+			$this->warning( sprintf( 'File does not exist: %s', $file ) );
+			return;
+		}
+
+		$contents = file_get_contents( $file ); // phpcs:ignore WordPress.WP.AlternativeFunctions
+
+		$pattern = '/' . preg_quote( $inner_pattern, '/' ) . '/';
+		if ( ! preg_match( $pattern, $contents, $matches, PREG_OFFSET_CAPTURE ) ) {
+			$this->warning( sprintf( 'Inner pattern %s not found in %s', $inner_pattern, $file ) );
+			return;
+		}
+
+		$offset  = $matches[0][1];
+		$length  = strlen( $contents );
+		$bracket = 0;
+
+		// Parse until we find beginning of statement pattern.
+		$start = strrpos( $contents, $statement_pattern, ( $length - $offset ) * -1 );
+
+		// Parse until we encounter closing bracket.
+		for ( $end = $offset; $end < $length; $end++ ) {
+			if ( '{' === $contents[ $end ] ) {
+				$bracket++;
+			}
+			if ( '}' === $contents[ $end ] ) {
+				$bracket--;
+				if ( 1 > $bracket ) {
+					break;
+				}
+			}
+		}
+
+		if ( false === $start || 0 > $start || $end >= $length ) {
+			$this->warning( sprintf( 'Statement %s not found in %s', $statement_pattern, $file ) );
+			return;
+		}
+
+		// Include whitespaces before start.
+		while ( 0 < $start && ctype_space( $contents[ $start - 1 ] ) ) {
+			$start--;
+		}
+
+		$new  = substr( $contents, 0, $start );
+		$new .= substr( $contents, $end + 1 );
+
+		if ( empty( $new ) ) {
+			$this->warning( sprintf( 'Replace failed for statement %s in %s', $statement_pattern, $file ) );
+			return;
+		}
+
+		file_put_contents( $file, $new ); // phpcs:ignore WordPress.WP.AlternativeFunctions
+	}
+
+	/**
+	 * Remove a file.
+	 *
+	 * @param string $file
+	 */
+	protected function remove_file( string $file ) {
+		if ( ! file_exists( $file ) ) {
+			$this->warning( sprintf( 'File does not exist: %s', $file ) );
+			return;
+		}
+
+		unlink( $file );
+	}
+
+	/**
+	 * Find a specific filename pattern within the polyfill folder.
+	 *
+	 * @param string $pattern Regexp pattern to match.
```

Comment:
I realise that this is copied from the other file, but technically this would be a glob pattern and not a regexp.